### ISSUE TITLE
Sverchok integration example

### ISSUE THREAD

**Issue by nortikin** [nomelif/Audionodes#30]
Cricket visualised with Sverchok addon.
You will need latest sverchok.
![изображение](https://user-images.githubusercontent.com/5783432/99663472-bb7df100-2a77-11eb-8324-5ded12e49361.png)

[audio-nodes-update2.blend.zip](https://github.com/nomelif/Audionodes/files/5566558/audio-nodes-update2.blend.zip)

**Comment by ollpu** [comment]
I'm working on an inspector node, which displays audio/other data in Blender. I'll see if I can make it affect a custom property, but you won't be able to access all of the audio data of course.

**Comment by nortikin** [comment]
> I'm working on an inspector node, which displays audio/other data in Blender. I'll see if I can make it affect a custom property, but you won't be able to access all of the audio data of course.

it would be great for generative design.

**Comment by enzyme69** [comment]
Yes would be great to have realtime audio nodes IO that can be hooked into SV or AnimationNodes. 

Like something below, I want to trigger animation into 3 different stream to visualise the audio and bake them. I also want to randomize while keeping the BPM. Don't know how... :) 

<img width="1093" alt="Screen Shot 2020-12-09 at 11 09 04 am" src="https://user-images.githubusercontent.com/3952687/101556466-05228180-3a0f-11eb-98c8-9871ef03b0d0.png">
